akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala

```diff
@@ -170,6 +170,14 @@ object ActorSystem {
     val appConfig = config.getOrElse(ConfigFactory.load(cl))
     createInternal(name, guardianBehavior, guardianProps, Some(appConfig), classLoader, executionContext)
   }
+  /**
+   * Scala API: Create an ActorSystem
+   */
+  def apply[T](
```

Comment:
We should / could make the complicated constructure private and just offer nice ones like this